core.md

```diff
@@ -57,23 +57,17 @@ You may [apply to join some teams](https://links.neoforged.net/apply), but some
 
 #### Steering Council
 
-The Steering Council is the overall management and oversight of the Project.
+The Steering Council are the representatives of the Project, final check for if a Project member is to be demoted or banned, and tiebreakers for perfectly split votes.
 
-They “steer” the project to keep it aligned with the long-term vision, hence its secondary name of Vision Team.
+The Council has at all times an odd number of active members, such that stalemates are not possible when making decisions. The members of the Council are decided by an annual internal vote to ensure fairness and cycling of council members over time. Details on the Steering Council voting process can be found at [/steering-council-voting.md](/steering-council-voting.md).
```

Comment:
For now I would keep the part about one new SC must be elected. We can discuss the whole topic in a separate PR that also outlines term limits (probably the upcoming one about elections in general)